src/plots/cartesian/layout_defaults.js

```diff
@@ -115,18 +115,34 @@ module.exports = function supplyLayoutDefaults(layoutIn, layoutOut, fullData) {
 
     var bgColor = Color.combine(plot_bgcolor, layoutOut.paper_bgcolor);
 
-    var axLayoutIn, axLayoutOut;
+    var axName, axLayoutIn, axLayoutOut;
 
     function coerce(attr, dflt) {
         return Lib.coerce(axLayoutIn, axLayoutOut, layoutAttributes, attr, dflt);
     }
 
-    axesList.forEach(function(axName) {
```

Comment:
There are still [two](https://github.com/plotly/plotly.js/pull/1431/files#diff-1602f9e66d27cf60c7469f31f03b8053R127) [maps](https://github.com/plotly/plotly.js/pull/1431/files#diff-1602f9e66d27cf60c7469f31f03b8053R136) and a [filter](https://github.com/plotly/plotly.js/pull/1431/files#diff-1602f9e66d27cf60c7469f31f03b8053R133) 🐎